CONTRIBUTING.md

```diff
@@ -150,6 +150,27 @@ We're currently experimenting with hosting the majority of lodestar packages and
   - Error: One or more main functionalities are not working, preventing some functions from working properly.
   - Fatal: One or more main functionalities are not working and preventing the application from fulfilling its duties.
 
+## Logging policy
```

Comment:
maybe add some text after this heading to explain in one sentence why the Logging policy section exists and what it is supposed to achieve

Reply:
kinda like this `Contributors must chose the log level carefully to ensure a consistent experience for every type of user`, maybe should just add this here and rephrase it a bit